src/components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -197,123 +180,154 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
-                onChange={handleChange}
-                optionLabel={(option) => option}
-              />
-            </div>
-            <div className="md:col-span-1">
-              <div className="">
-                {assignedUserLoading ? (
-                  <CircularProgress />
-                ) : (
-                  <UserAutocomplete
-                    label="Assigned To"
-                    value={assignedUser === null ? undefined : assignedUser}
-                    onChange={handleOnSelect}
-                    error=""
-                    name="assigned_to"
+          <Form
+            className=""
+            defaults={state.form}
+            onSubmit={handleSubmit}
+            onCancel={() => goBack()}
+            noPadding
+            hideRestoreDraft
+          >
+            {(field) => (
+              <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
+                <div className="md:col-span-1">
+                  <SelectFormField
+                    label="Status"
+                    name="status"
+                    value={field("status").value}
+                    options={resourceStatusOptions}
+                    onChange={(e) => field("status").onChange(e)}
+                    optionLabel={(option) => option}
+                  />
+                </div>
+                <div className="md:col-span-1">
+                  <div className="">
+                    {assignedUserLoading ? (
+                      <CircularProgress />
+                    ) : (
+                      <UserAutocomplete
+                        label="Assigned To"
+                        value={field("assigned_to_object").value}
+                        onChange={(e) => {
+                          field("assigned_to_object").onChange({
+                            name: "assigned_to_object",
+                            value: e.value,
+                          });
+                          field("assigned_to").onChange({
+                            name: "assigned_to",
+                            value: e.value.id,
+                          });
+                        }}
+                        error=""
+                        name="assigned_to"
+                      />
+                    )}
+                  </div>
+                </div>
+                <div>
+                  <FieldLabel>Name of resource approving facility</FieldLabel>
+                  <FacilitySelect
+                    multiple={false}
+                    name="approving_facility"
+                    facilityType={1500}
+                    selected={field("approving_facility_object").value}
+                    setSelected={(obj: any) => {
+                      field("approving_facility_object").onChange({
+                        name: "approving_facility_object",
+                        value: obj,
+                      });
+                      field("approving_facility").onChange({
```

Comment:
What does this `approving_facility` correspond to? 🤔  We already have the object one and that should be enough, no?